packages/hub/services/serializers/scheduled-payment-attempt-serializer.ts

```diff
@@ -0,0 +1,52 @@
+import { ScheduledPaymentAttemptWithScheduledPayment } from '../../routes/scheduled-payment-attempts';
+import { JSONAPIDocument } from '../../utils/jsonapi-document';
+import { uniqBy } from 'lodash';
+import ScheduledPaymentSerializer from './scheduled-payment-serializer';
+import { inject } from '@cardstack/di';
+
+export default class ScheduledPaymentAttemptSerializer {
+  scheduledPaymentSerializer: ScheduledPaymentSerializer = inject('scheduled-payment-serializer', {
+    as: 'scheduledPaymentSerializer',
+  });
+
+  serialize(
+    model: ScheduledPaymentAttemptWithScheduledPayment | ScheduledPaymentAttemptWithScheduledPayment[]
+  ): JSONAPIDocument {
+    if (Array.isArray(model)) {
+      return {
+        data: model.map((model) => this.serialize(model).data),
+        included: uniqBy(model, 'scheduledPayment.id').map((model) => {
```

Comment:
Include scheduled payment as a related model in JSON:API fashion - https://jsonapi.org/format/#document-compound-documents